samples/MobileBuyIntegration/app/src/main/java/com/shopify/checkout_sdk_mobile_buy_integration_sample/cart/data/CartRepository.kt

```diff
@@ -27,29 +27,31 @@ import com.shopify.buy3.Storefront.CartLineInput
 import com.shopify.checkout_sdk_mobile_buy_integration_sample.cart.data.source.network.CartStorefrontApiClient
 import com.shopify.checkout_sdk_mobile_buy_integration_sample.common.ID
 import com.shopify.checkout_sdk_mobile_buy_integration_sample.common.toGraphQLId
+import timber.log.Timber
 import kotlin.coroutines.suspendCoroutine
 
 class CartRepository(
     private val cartStorefrontApiClient: CartStorefrontApiClient,
 ) {
 
-    suspend fun createCart(variantId: ID, quantity: Int, demoBuyerIdentityEnabled: Boolean): CartState.Cart {
+    suspend fun createCart(
+        variantId: ID,
+        quantity: Int,
+        demoBuyerIdentityEnabled: Boolean,
```

Comment:
Might be nice to maintain a static "AppConfig" object with this value on it and just look at the config inside this function